src/sentry/api/validators/__init__.py

```diff
@@ -7,6 +7,5 @@
 from .notifications import *  # noqa: F401,F403
 from .project import *  # noqa: F401,F403
 from .project_codeowners import *  # noqa: F401,F403
-from .sentry_apps.schema import *  # noqa: F401,F403
```

Comment:
this star import clobbered a function with a different definition.  as far as I can tell noone relied on this star import anyway!